cmd/tink-cli/cmd/template/create.go

```diff
@@ -2,36 +2,41 @@ package template
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"log"
 	"os"
-	tt "text/template"
 
 	"github.com/spf13/cobra"
 	"github.com/tinkerbell/tink/client"
 	"github.com/tinkerbell/tink/protos/template"
+	"github.com/tinkerbell/tink/workflow"
 )
 
 var (
-	fPath        = "path"
-	fName        = "name"
-	filePath     string
-	templateName string
+	file     = "file"
+	filePath string
```

Comment:
I don't quite see the reason for `file` global var when its not used in the `flags.StringVarP` call. It should either be named a little more descriptively (filePathArgName?) and used there too or we just hard code use of `"file"` every where.

I'm not a fan of the "variable for the variable name" approach...